modules/katello_manifest.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2017, Andrew Kofink <ajkofink@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: katello_manifest
+short_description: Create and Manage Katello manifests
+description:
+    - Create and Manage Katello manifests
+author: "Andrew Kofink (@akofink)"
+requirements:
+    - "nailgun >= 0.29.0"
+    - "python >= 2.6"
+options:
+    server_url:
+        description:
+            - URL of Foreman server
+        required: true
+    username:
+        description:
+            - Username on Foreman server
+        required: true
+    password:
+        description:
+            - Password for user accessing Foreman server
+        required: true
+    verify_ssl:
+        description:
+            - Verify SSL of the Foreman server
+    organization:
+        description:
+            - Organization that the manifest is in
+        required: true
+    manifest_path:
+        description:
+            - Path to the manifest zip file
+    state:
+        description:
+            - The state of the manifest
+        required: true
+        choices:
+            - absent
+            - present
+            - refreshed
+    repository_url:
```

Comment:
I am still  of the opinion that this is confusing given its talked about more so as the CDN URL If you feel this needs to match the API param then lets change it to `redhat_repository_url` which I think is closer to the fact that its a CDN URL (or CDN like URL in the case of disconnected).

The UI shows it as "Red Hat CDN URL" and the CLI as "redhat-repository-url"